README.md

```diff
@@ -257,6 +257,12 @@ interface WowerlayProps {
    * @default false
    */
   syncHeight?: boolean;
+
+  /**
+   * Any given attribute (except key and ref) is passed to Wowerlay`s wrapper background element.
+   * @default {}
+   */
+  backgroundAttrs?: Record<string | number, any>;
```

Comment:
Maybe we can also say ts to exclude these keys:

```js
type BackgroundAttrs = {
    [key: string | number]: any,
} & {
    ref?: never,
    key?: never
}
```

Reply:
Typescript treats `never` as `undefined` in that scenario, I updated it to `undefined | null`